From 9cc7ada1d3e6242c0ca2673277eb4d1c5dc281e8 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Thu, 18 Apr 2024 14:55:50 +0100
Subject: [PATCH 1032/1085] fixup! bcm2708 framebuffer driver

Make CONFIG_FB_BCM2708 select CONFIG_FB_DEVICE, to avoid potential build
problems.

See: https://github.com/raspberrypi/linux/issues/6059

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 drivers/video/fbdev/Kconfig | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/video/fbdev/Kconfig
+++ b/drivers/video/fbdev/Kconfig
@@ -64,6 +64,7 @@ config FB_MACMODES
 config FB_BCM2708
 	tristate "BCM2708 framebuffer support"
 	depends on FB && RASPBERRYPI_FIRMWARE
+	select FB_DEVICE
 	select FB_CFB_FILLRECT
 	select FB_CFB_COPYAREA
 	select FB_CFB_IMAGEBLIT
